### ISSUE TITLE
[Bug]: Cannot install PWA (Android 13, Firefox + Chrome)

### ISSUE THREAD

**Issue by Nonononoki** [MarmadileManteater/FreeTubeCordova#297]
### Guidelines

- [X] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the bug report I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [X] This issue contains only one bug.

### Describe the bug

There is currently no option to install the PWA.

### Expected Behavior

There should be an option to install the PWA, otherwise it's just a website.

### Issue Labels

accessibility issue

### FreeTube Version

v0.19.1.109 Beta 

### Operating System Version

Android 13

### Installation Method

PWA (Progressive Web App)

### Primary API used

Local API

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by MarmadileManteater** [comment]
I noticed this recently too. I'm not sure if the requirements for manifests changed. I haven't looked into it more than having verified that it doesn't show the option to install anymore despite the fact that it used to, and I don't think anything changed on freetube's side.

**Comment by MarmadileManteater** [comment]
I believe the issue was that I was providing the incorrect app icon sizes. The PWA should now be installable in the latest nightly.

**Comment by MarmadileManteater** [comment]
This issue should be solved by the latest release.
